bridge-fungible-token/src/utils.sw

```diff
@@ -19,14 +28,61 @@ const GTF_INPUT_MESSAGE_DATA = 0x11E;
 const GTF_INPUT_MESSAGE_SENDER = 0x115;
 const GTF_INPUT_MESSAGE_RECIPIENT = 0x116;
 
+fn bn_mul(bn: U256, d: u64) -> (b256, u64) {
+    disable_panic_on_overflow();
+    let tuple_result: (b256, u64) = (
+        0x0000000000000000000000000000000000000000000000000000000000000000,
+        0,
+    );
+    let product_buffer = asm(bn: __addr_of(bn), d: d, c0, c1, v, f, product, carry_offset, product_buffer: __addr_of(tuple_result)) {
```

Comment:
the `f` register was used to remember the flag values, but you removed that part and now it's not used.